### PR TITLE
Add October Monthly Update

### PR DESCRIPTION
**Pull request by lauragift21**
- Add October monthly update
- Minor grammar fixes

### CODE REVIEW

File: _posts/2020-11-16-october-rubygems-update.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+* improved the deprecation path for gem query. - [#4021](https://github.com/rubygems/rubygems/pull/4021)
+* fixed issues with Bundler not loading RubyGems plugins in `$LOAD_PATH` locations, which was affecting some version managers. - [#3534](https://github.com/rubygems/rubygems/pull/3534)
+
+As always, we continue responding to RubyGems & Bundler issues and PRs, doing issue triage on both old and new issues, and reducing the number of open tickets.
+
+In October, Rubygems gained [143 new commits](https://github.com/rubygems/rubygems/compare/master@%7B2020-10-01%7D...master@%7B2020-10-31%7D), contributed by 11 authors. There were 2889 additions and 1705 deletions across 889 files.
+
+## RubyGems.org News
+
+This month we implemented a [bulk update to RubyGem downloads count](https://github.com/rubygems/rubygems.org/pull/2527) to reduce the processing time for `FastlyLogProcessor` by about 95%, up to 40 seconds. We studied an old HackerOne Report and proposed a solution for it that will get implemented into RubyGems.org.
```
Comment (lauragift21) [inline _posts/2020-11-16-october-rubygems-update.md:31]: Hi @sonalkr132, Can you help clarify this sentence? What was the previous processing time for example?
```we implemented a [bulk update to RubyGem downloads count](https://github.com/rubygems/rubygems.org/pull/2527) to reduce the processing time for `FastlyLogProcessor` by about 95%, up to 40 seconds. ```

File: _posts/2020-11-16-october-rubygems-update.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+title: October 2020 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Together](http://rubytogether.org/), we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems, Ruby Toolbox and RubyGems.org in September.
+
+## RubyGems News
+
+In October we released a new Bundler version --- **`2.2.0.rc.2`**. We also upgraded Bundler's & Rubygems' vendored Molinillo to **`0.7.0`**  (it's latest release). - [#3402](https://github.com/rubygems/rubygems/pull/3402), [#3388](https://github.com/rubygems/rubygems/pull/3388).
```
Comment (deivid-rodriguez) [inline _posts/2020-11-16-october-rubygems-update.md:11]: I'd highlight "In October, we released Bundler 2.2.0.rc.2 and rubygems 3.2.0.rc.2", and move the Molinillo upgrade to the list of other improvements.

File: _posts/2020-11-16-october-rubygems-update.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Together](http://rubytogether.org/), we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems, Ruby Toolbox and RubyGems.org in September.
+
+## RubyGems News
+
+In October we released a new Bundler version --- **`2.2.0.rc.2`**. We also upgraded Bundler's & Rubygems' vendored Molinillo to **`0.7.0`**  (it's latest release). - [#3402](https://github.com/rubygems/rubygems/pull/3402), [#3388](https://github.com/rubygems/rubygems/pull/3388). 
+
+We also made the following changes and improvements:
+* fixed an [issue with the `specific_platform` setting.](https://github.com/rubygems/rubygems/pull/4022) 
+* merged a PR that stops changing the CWD for building extensions, which should allow concurrent extension compilation without any contention.
+* worked on PRs to improve independence between test and lib code (simplifies and helps packagers), and changes to improve specs.
```
Comment (deivid-rodriguez) [inline _posts/2020-11-16-october-rubygems-update.md:16]:
```suggestion
* worked on PRs to improve independence between test and lib code (simplifies and helps packagers), and added some changes to improve specs.
```

File: _posts/2020-11-16-october-rubygems-update.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+* added improvements to require more default gems lazily.
+* fixed an issue with help commands when Bundler installed from `ruby-core` installer and `man` are not available. - [#3997](https://github.com/rubygems/rubygems/pull/3997)
```
Comment (deivid-rodriguez) [inline _posts/2020-11-16-october-rubygems-update.md:18]:
```suggestion
* fixed an issue with help commands when Bundler has been installed by `ruby-core` installer and `man` is not available. - [#3997](https://github.com/rubygems/rubygems/pull/3997)
```

File: _posts/2020-11-16-october-rubygems-update.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Together](http://rubytogether.org/), we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems, Ruby Toolbox and RubyGems.org in September.
+
+## RubyGems News
+
+In October we released a new Bundler version --- **`2.2.0.rc.2`**. We also upgraded Bundler's & Rubygems' vendored Molinillo to **`0.7.0`**  (it's latest release). - [#3402](https://github.com/rubygems/rubygems/pull/3402), [#3388](https://github.com/rubygems/rubygems/pull/3388). 
+
+We also made the following changes and improvements:
+* fixed an [issue with the `specific_platform` setting.](https://github.com/rubygems/rubygems/pull/4022) 
+* merged a PR that stops changing the CWD for building extensions, which should allow concurrent extension compilation without any contention.
```
Comment (deivid-rodriguez) [inline _posts/2020-11-16-october-rubygems-update.md:15]: I guess we should link to the PR here too?

```suggestion
* merged a PR that stops changing the CWD for building extensions, which should allow concurrent extension compilation without any contention. - [#3498](https://github.com/rubygems/rubygems/pull/3498)
```

**Review by deivid-rodriguez — COMMENTED** [review]
Thanks @lauragift21! Added a few small tweaks but it's ready from my side otherwise :)

File: _posts/2020-11-16-october-rubygems-update.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+## RubyGems.org News
+
+This month we implemented a [bulk update to RubyGem downloads count](https://github.com/rubygems/rubygems.org/pull/2527) to reduce the processing time for `FastlyLogProcessor` by about 95%, up to 40 seconds. We studied an old HackerOne Report and proposed a solution for it that will get implemented into RubyGems.org. 
+
+We also made the following changes and improvements:
+
+* imported Fastly rubygems.org production configuration to Terraform. - [#4](https://github.com/rubygems/rubygems-terraform/pull/4) 
+* updated Fastly `vcl` to unset X-Forwarded-Host from requests to fix a HO report. - [#5](https://github.com/rubygems/rubygems-terraform/pull/5)
+* added regex whitelist for URL on honeycomb logs export to ensure we don't inadvertently send any sensitive information. - [#7](https://github.com/rubygems/rubygems-terraform/pull/7)
```
Comment (sonalkr132) [inline _posts/2020-11-16-october-rubygems-update.md:37]: I am not sure if we are adding any value by adding links to private repo here. I link https://github.com/rubygems/rubygems-terraform in my task updates as some people in the channel may have access.

File: _posts/2020-11-16-october-rubygems-update.md (hunk 1 above)
Comment (lauragift21) [reply _posts/2020-11-16-october-rubygems-update.md:31]: Alright, I think that's a better way of phrasing it.

File: _posts/2020-11-16-october-rubygems-update.md (hunk 2 above)
Comment (lauragift21) [reply _posts/2020-11-16-october-rubygems-update.md:11]: Thank you @deivid-rodriguez.

File: _posts/2020-11-16-october-rubygems-update.md (hunk 6 above)
Comment (lauragift21) [reply _posts/2020-11-16-october-rubygems-update.md:37]: True. I'll take the links out just mentioning the task works as well.

**Review by sonalkr132 — APPROVED** [review]

**Comment by deivid-rodriguez** [comment]
Thank you @lauragift21! :tada:

**Pull request merged**